internal/appsec/dyngo/instrumentation/httpsec/tags.go

```diff
@@ -72,16 +72,27 @@ func init() {
 	sort.Strings(collectedHTTPHeaders[:])
 }
 
-// SetSecurityEventTags sets the AppSec-specific span tags when a security event occurred into the service entry span.
-func SetSecurityEventTags(span instrumentation.TagSetter, events []json.RawMessage, headers, respHeaders map[string][]string) {
+// setSecurityEventsTags sets the AppSec-specific span tags when a security event occurred into the service entry span.
+func setSecurityEventsTags(span instrumentation.TagSetter, events []json.RawMessage) {
 	if err := instrumentation.SetEventSpanTags(span, events); err != nil {
-		log.Error("appsec: unexpected error while creating the appsec event tags: %v", err)
+		log.Error("appsec: unexpected error while creating the appsec events tags: %v", err)
 	}
+}
+
+// setRequestHeadersTags sets the AppSec-specific request headers span tags.
+func setRequestHeadersTags(span instrumentation.TagSetter, headers map[string][]string) {
+	setHeadersTags(span, "http.request.headers.", headers)
```

Comment:
Same question about wrapping this. Can't we simply use this oneliner in `WrapHandler`?

Reply:
For this one and the response equivalent, I find the code calling them more readable and less error prone.